### PR TITLE
Update vlucas/phpdotenv to v5.3

### PR DESCRIPTION
**Pull request by ottok**
Changes listed at:
https://github.com/vlucas/phpdotenv/blob/master/UPGRADING.md

Priority: low

Pending for @elguitar to take over this.

### CODE REVIEW

**Comment by ottok** [comment]
Tested with:
```
cat .env 
OTTO=ottowashere
```

and
```
<?php echo getenv('OTTO'); ?>
```

in a theme file, and it did output the env and inherited it correctly from the `.env` file

File: htdocs/wp-config.php [hunk 1]
```diff
@@ -13,14 +13,16 @@
...
+ * Seravo provides all needed envs for WordPress by default. For details see
+ * https://seravo.com/docs/development/environment-variables/#using-dotenv
+ *
+ * The syntax below is written for Dotenv 5.x. For changes to previous versions
+ * see  https://github.com/vlucas/phpdotenv/blob/master/UPGRADING.md
  */
 if ( file_exists($root_dir . '/.env') ) {
-  $dotenv = new Dotenv\Dotenv($root_dir);
-  $dotenv->overload();
+  $dotenv = Dotenv\Dotenv::createUnsafeImmutable($root_dir);
```
Comment (elguitar) [inline htdocs/wp-config.php:24]: This seems unsafe. However, this change is listed in https://github.com/vlucas/phpdotenv/blob/master/UPGRADING.md, so I believe that this is the safest way to go for now.

**Review by elguitar — APPROVED** [review]
Looks good to me!

Having something called `createUnsafeImmutable` does not strike to me as the most stable solution, but for now it is okay.

**Comment by ottok** [comment]
If we don't do that change, then none of the `getenv()` calls would
work, and the whole .env idea would be kind of moot. At least to my
understanding, I could have missed something as well.

**Comment by ottok** [comment]
Docs updated in https://github.com/Seravo/docs/commit/27076f0b3e507cfe6c9080f3083b20df98fab037

**Pull request merged**